web/src/components/snapshots/ConfigureSnapshots.jsx

```diff
@@ -41,6 +74,9 @@ class ConfigureSnapshots extends React.Component {
       isKurlEnabled,
     } = this.props;
 
+    const fsBackupComponentName = this.getFSBackupComponentName();
+    const fsBackupComponentFlags = this.getFSBackupComponentFlags();
+
```

Comment:
```suggestion
```
good to avoid temp variables if there is no renaming going on. function calls are cheap